pkgs/development/tools/misc/unixbench/default.nix

```
@@ -0,0 +1,111 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, makeWrapper
+, pandoc
+, installShellFiles
+, perl
+, xorg
+, libGL
+, bash
```

```suggestion
, runtimeShell
```
replace "/bin/sh"  to  "${runtimeShell}"